src/components/FilterOptions.tsx

```diff
@@ -22,7 +22,7 @@ const FilterOptions = (props: Types.FilterOptionsProps) => {
       <form className="visibility">
         <input
           type="checkbox"
-          onClick={EventHandlers.handleVisibilityToggle(props.stateManager)}
+          onChange={EventHandlers.handleVisibilityToggle(props.stateManager)}
```

Comment:
Related to "Resolved "failed prop type check" by changing onClick to onChange for checkbox"